lab-sarah/server.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+//node modules
+const url = require('url');
+const http = require('http');
+const queryString = require('querystring');
+
+//npm modules
+const cowsay = require('cowsay');
+
+//module constants
+const PORT = process.env.PORT || 3000;
+
+const parseBody = require('./lib/parse-body.js');
+
+//module logic
+const server = http.createServer(function(req, res) {
+  // console.log('BEGINNING', res, 'END OF RESPONSE');
```

Comment:
Please remove console.logs and comments prior to submission unless they're relevant to your project (such as logging an error to the server).